code/src/java/pcgen/cdom/meta/ConvertingFacetView.java

```diff
@@ -65,25 +64,18 @@ public String getDescription()
 	}
 
 	@Override
-	public boolean represents(Object src)
```

Comment:
Unfortunately, you are now altering code that has private branches trying to use such code.  Please don't remove this from the interface.